script/generate_artifact.sh

```diff
@@ -2,4 +2,5 @@
 mkdir bin
 cp target/release/{plume,plm} bin
 strip -s bin/*
-tar -cvzf plume.tar.gz bin/ static/ migrations/$FEATURES
+tar -cvzf plume.tar.gz bin/ static/
+tar -cvzf wasm.tar.gz static/plume-front.{js,wasm}
```

Comment:
This one, should contain all of `static/` — it's entirely architecture independent!

Reply:
I expected it to be an alternative step to all `cargo web` things in [here](https://docs.joinplu.me/installation/with/source-code), in which case it doesn't require the whole `static/` dir as it is `git clone`d. What were your expectations?

Reply:
you are right.